rules/cloudtrail/cloudtrail_critical_api.py

```diff
@@ -0,0 +1,41 @@
+from stream_alert.rule_processor.rules_engine import StreamRules
+from rules.helpers.base import in_set
+
+rule = StreamRules.rule
+disable = StreamRules.disable()
+
+
+@rule(logs=['cloudtrail:api_events'],
+      matchers=[],
+      outputs=['aws-s3:sample.bucket'])
```

Comment:
I have a bunch of these I'm fixing now in other rules - please adhere to what jack is suggesting with `aws-s3:sample_bucket`